TestAssets/TestProjects/LightupLib/project.json

```diff
@@ -0,0 +1,386 @@
+{
```

Comment:
I have added a test that brings in dependency of LightupLib and also looks up lightup deps.json from shared location.